auth-foundation/src/main/java/com/okta/authfoundation/AuthFoundation.kt

```diff
@@ -22,4 +22,8 @@ object AuthFoundation {
     fun initializeAndroidContext(context: Context) {
         ApplicationContextHolder.setApplicationContext(context)
     }
+
+    internal suspend fun initializeStorage() {
+        AuthFoundationDefaults.tokenStorageFactory()
```

Comment:
So you're touching the factory and ignoring its response object to ensure that it is initialized?

Reply:
Yeah exactly